packages/ui/Marquee/MarqueeDecorator.js

```diff
@@ -226,9 +226,7 @@ const MarqueeDecorator = hoc(defaultConfig, (config, Wrapped) => {
 			/**
 			 * Determines what triggers the marquee to start its animation.
 			 *
-			 * Valid values are `'focus'`, `'hover'` and `'render'`. The default is `'focus'`.
-			 *
-			 * @type {String}
+			 * @type {('focus'|'hover'|'render')}
```

Comment:
Do we render out the possible values of typescript in our docs for cases like this? If so, neat!

Reply:
Something like that.  We do render the values into our typescript def. files and we do render out the array of possible values into the docs.  I think it's much more descriptive than just 'String'

Reply:
It is indeed much more descriptive. I'm just verifying that we're not _losing_ documentation/description on our website. Whether, for example, someone who doesn't use typescript would be able to read the list of acceptable values in some way.

Reply:
Nope, we don't lose anything by this.  The docs will list the acceptable values.

Reply:
docs
![image](https://user-images.githubusercontent.com/788456/61483173-23d33480-a951-11e9-8ea1-d5102c382986.png)

.d.ts
![image](https://user-images.githubusercontent.com/788456/61483200-2fbef680-a951-11e9-9916-77fd0ead8416.png)

codesandbox.io
![image](https://user-images.githubusercontent.com/788456/61483537-d86d5600-a951-11e9-97d5-7e8c00359f03.png)

Reply:
Cool. Thanks for sharing the pictures. They look different from what I was seeing.